orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -24,6 +25,18 @@
 # Needed to re-implement PyTorch's cpu,cuda,to methods
 T = TypeVar('T', bound='Module')
 
+torch_cuda_allocator_addresses_cpp_source = """
+#include <torch/extension.h>
+#include <c10/cuda/CUDACachingAllocator.h>
+
+size_t cuda_caching_allocator_raw_alloc_address() {
+    return reinterpret_cast<size_t>(&c10::cuda::CUDACachingAllocator::raw_alloc);
+}
+
+size_t cuda_caching_allocator_raw_delete_address() {
+    return reinterpret_cast<size_t>(&c10::cuda::CUDACachingAllocator::raw_delete);
+}
+"""
```

Comment:
How about moving this constant along with `load_inline()` call below to a helper function outside ortmodule.py? 
This is low level implementation detail that is better off away from the main python module file